cicerone/spec/operation.py

```diff
@@ -0,0 +1,49 @@
+"""Operation model for HTTP operations."""
+
+from typing import Any, ClassVar, Mapping
+
+from pydantic import BaseModel, Field
+
+
+class Operation(BaseModel):
+    """Represents an HTTP operation (GET, POST, etc.)."""
+
+    model_config = {"extra": "allow"}
+
+    # Fields that are explicitly handled
+    HANDLED_FIELDS: ClassVar[set[str]] = {"operationId", "summary", "description", "tags", "parameters", "responses"}
```

Comment:
It is not clear what this is used for - does it need a clearer name? It feels like it is just because they are all easy strs - perhaps it needs a name appropriate for that.